scss/tech.scss

```diff
@@ -1,55 +1,61 @@
 $tech-box-border-radius: 10px;
-$tech-box-height: 450px;
+$tech-box-height: 480px;
 $tech-box-width: 300px;
 $tech-logo-size: 100px;
 $tech-text-margin: 20px;
 
-.tech-box {
-  background-color: $green;
-  border: 0;
-  border-radius: $tech-box-border-radius;  
+.tech-box-list {
   display: flex;
-  flex-direction: column;
-  height: $tech-box-height;
-  margin: 35px;
-  padding: 40px;
-  position: relative;
-  text-align: center;
-  width: $tech-box-width;
+  flex-wrap: wrap;
+  padding-left: 50px;
```

Comment:
Try to make paddings and margins as symmetric as possible. This can help you handle LTR and RTL writing systems in the best way! 🙂